client/strings/en-us.json

```diff
@@ -96,6 +96,7 @@
   "HeaderAudiobookTools": "Audiobook File Management Tools",
   "HeaderAudioTracks": "Audio Tracks",
   "HeaderAuthentication": "Authentication",
+  "HeaderCustomMetadataProviders": "Custom metadata providers",
```

Comment:
These should be alphabetical (ignoring casing). Also, don't worry about adding the strings to the other languages, I have a local script I run for that.